### PR TITLE
Adopt localized resources

### PR DESCRIPTION
**Pull request by abdalalii1**
As of issue https://github.com/sindresorhus/Preferences/issues/77 i localized package strings in a Resource Folder exactly as described in Apple documentation.

PS: while I was trying to sync the fork the initial PR was intentionally merged.

### CODE REVIEW

File: Sources/Preferences/SettingsTabViewController.swift [hunk 1]
```diff
@@ -95,9 +95,9 @@ final class SettingsTabViewController: NSViewController, SettingsStyleController
 			} else {
 				let settings: String
 				if #available(macOS 13, *) {
-					settings = Localization[.settings]
+                    			settings = NSLocalizedString("settings", bundle: .module, comment: "")
```
Comment (sindresorhus) [inline Sources/Preferences/SettingsTabViewController.swift:98]: Still incorrect indentation.

Comment (abdalalii1) [reply Sources/Preferences/SettingsTabViewController.swift]: are you using a different indentation configuration? to align, what are yours?!

indentation

**Comment by ShikiSuen** [comment]
Please use these lang tags for Chinese:

zh-Hans in lieu of zh-CN.
zh-Hant in lieu of zh-TW.

File: Sources/Preferences/SettingsTabViewController.swift (hunk 1 above)
Comment (sindresorhus) [reply Sources/Preferences/SettingsTabViewController.swift]: You are asking questions that you can simply check yourself. The existing indentation exists in all the source files.

**Comment by sindresorhus** [comment]
Closing for lack of activity. Going to leave this for someone else.

**Comment by DivineDominion** [comment]
@ShikiSuen Could you apply the changes you proposed and open a new PR?

**Comment by ShikiSuen** [comment]
@DivineDominion I'll see whether I can find some time to do this in this month (Jan).

**Comment by ShikiSuen** [comment]
@DivineDominion I'll do this after #90 gets merged, reducing possibilities of merging conflicts.

**Pull request closed without merging**